controller/src/main/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessors.java

```diff
@@ -241,14 +242,14 @@ public boolean isReady() {
 
     private CompletableFuture<Void> createScope(final String scopeName) {
         return Futures.toVoid(Retry.indefinitelyWithExpBackoff(DELAY, MULTIPLIER, MAX_DELAY,
-                e -> log.warn("Error creating event processor scope " + scopeName, e))
+                e -> log.warn("Error creating event processor scope {} with exception {}", scopeName, Exceptions.unwrap(e).getClass().getName()))
```

Comment:
Is `getName()` maybe leaving out too much info? Do you prefer that to the `toString()` result of the unwrapped exception?